procrastinate/retry.py

```diff
@@ -31,7 +31,7 @@ def get_schedule_in(self, attempts: int) -> Optional[float]:
         ----------
         attempts:
             The number of previous attempts for the current job. The first time
-            a job is ran, `attempts` will be 0.
+            a job is run, `attempts` will be 0.
```

Comment:
:notes: And I raaaaan :notes:

Reply:
run, ran, run, je suis formelle :)
I run
I ran
I have run

Reply:
Je ne contredisais pas :> Je me rappelais les annees 80 :santa: